components/blitz/src/ome/formats/importer/cli/CommandLineImporter.java

```diff
@@ -342,6 +342,8 @@ public static void advUsage() {
             + "          ln              \t# Use hard-link.\n"
             + "          ln_s            \t# Use soft-link.\n"
             + "          ln_rm           \t# Caution! Hard-link followed by source deletion.\n\n"
```

Comment:
This and the next line need one '\n` removing. Only the last option needs the extra newline.